src/components/Grid/Grid.test.tsx

```diff
@@ -0,0 +1,30 @@
+// importing boilerplate stuff
+import React from 'react';
+import { composeStories } from '@storybook/testing-react';
+import { render } from '@testing-library/react';
+import * as stories from './Grid.stories';
+
+// importing my stories to test
+const { ResponsiveCardGrid, GridWithinGrid, GridPlayground } =
+    composeStories(stories);
+
+// /////////////////////////////////////////////////////
+// examples of basic tests of props, values and styles
+// /////////////////////////////////////////////////////
+
+// Test Story 1: Responsive card grid
+test('Renders Default component', () => {
+    render(<ResponsiveCardGrid />);
+});
```

Comment:
Extra Goal: It would be great if you can add tests to verify whether the functionality is working or not. 🙌🏻

Reply:
Can you give an example how it is done or tested.
I looked into other components `text.tsx` files, but i did not understand how the test functions are affecting the storybook or where the tests are running. 

-Thank You🙏

Reply:
Ahh, Go it. Found a youtube vid😁

Reply:
Cool, just some basic tests would be nice. TY 🙌🏻 
Check out our contributing guide if you need more help 😄 
https://github.com/web3ui/web3uikit/blob/master/CONTRIBUTE.md

Reply:
i think those docs are out of date for testing since React 18 @AbhinavMV 

have a look at `Accordion.test.tsx` for some simple test cases and setup @johnvsnagendra